packages/marshal/src/deeplyFulfilled.js

```diff
@@ -3,15 +3,14 @@
 /// <reference types="ses"/>
 
 import { E } from '@endo/eventual-send';
-/**
- * @template T
- * @typedef {import('@endo/eventual-send').ERef<T>} ERef
- */
 import { isPromise } from '@endo/promise-kit';
 import { getTag, isObject } from './helpers/passStyle-helpers.js';
 import { makeTagged } from './makeTagged.js';
 import { passStyleOf } from './passStyleOf.js';
 
+/** @typedef {import('./types').Passable} Passable */
```

Comment:
Nit: while tsc doesn't care as much, I think we should keep the habit of using fully qualified names, aka with extensions, for imports

Reply:
Reasons to leave it this way:
- There should never be any ambiguity what file it refers to. (There must not be a `types.ts` next to a `types.js`.)
- We can change the syntax of the imported file between `.ts` and `.js` without touching everywhere it's imported.

Reply:
Even if there is a `.d.ts` or `.ts` file, the import should always be of `.js` file per TypeScript conventions. TypeScript will internally use the `.d.ts` file if it exists. We use explicit `.js` extensions for our imports as automatic extension resolution is something we left behind when switching to native ESM.

Reply:
Thanks for the correction on the second bullet about changing extensions. I didn't know that TypeScript conventions are that if the file is `foo.ts` you should still import it as `foo.js`.

I neglected to include what I think is the most compelling reason to leave it this way: less to type and read. I think simplicity wins unless there is a compelling reason to add the extension. Is it to be consistent with the JSM module imports? I think consistency is important when it reduces confusion or misunderstanding, but I have a hard time imagining someone being confused by the absence of `.js` here in the JSDoc.

Reply:
Right I was looking for consistency. I've never found terseness to be very compelling. We require extensions for ESM imports (static, and in the future dynamic), and since TS style jsdoc type imports use a dynamic-like ESM notion, I figure we should stay consistent.